components/help/help_controller.tsx

```diff
@@ -20,8 +20,7 @@ type Props = {
 
 export default class HelpController extends React.PureComponent<Props> {
     public componentDidUpdate(): void {
-        // eslint-disable-next-line react/no-find-dom-node
-        const helpControllerNode = ReactDOM.findDOMNode(this);
+        const helpControllerNode = findDOMNode(this);
```

Comment:
We actually still want this warning because we shouldn't be using `findDOMNOde` any more. We had a HW campaign about it a while back to remove it, but there's a few spots we missed.

You can leave this as-is, and I'll just base my PR off this one since you already have some conflicts to resolve after I merged my `no-string-refs` one which was actually part of a similar campaign around the same time

Reply:
Got it reverted the change